### PR TITLE
add ward residency chips

### PR DESCRIPTION
**Pull request by theyokohamalife**
Adds chips to show if ward residency is required or not.

### How to test
Check Netlify preview.

### Screenshots
Screen Shot 2021-06-28 at 11 02 00 PM

### CODE REVIEW

File: components/CancellationList.vue [hunk 1]
```diff
@@ -43,6 +43,26 @@
           {{ $t("cancelList.voucher.notRequired") }}
         </v-chip>
       </template>
+      <template v-slot:[`item.wardResidencyRequired`]="{ item }" align="left">
+        <v-chip
+          v-if="item.wardResidencyRequired === true"
+          class="ma-2"
```
Comment (Ermish) [inline components/CancellationList.vue:49]: should move styling classes to css. 
see explanation #189

File: components/CancellationList.vue [hunk 2]
```diff
@@ -43,6 +43,26 @@
...
+          color="pink"
+          label
+          text-color="white"
+        >
+          {{ $t("cancelList.wardResidency.required") }}
+        </v-chip>
+        <v-chip
+          v-if="item.wardResidencyRequired === false"
+          class="ma-2"
```
Comment (Ermish) [inline components/CancellationList.vue:58]: should move styling classes in css. 
see explanation #189

**Review by Ermish — APPROVED** [review]
LGTM outside of the styling nits. :side-eye:

**Comment by theyokohamalife** [comment]
> LGTM outside of the styling nits. :side-eye:

After this feature I'll work on cleaning up the code.

**Pull request merged**